src/com/Jacksonnn/DCCore/DCCore.java

```diff
@@ -106,6 +109,24 @@ public void onEnable() {
 		BukkitScheduler scheduler = Bukkit.getServer().getScheduler();
 		scheduler.scheduleSyncRepeatingTask(this, new AnnouncerThread(this), AnnouncementManager.getInterval() * 20L, AnnouncementManager.getInterval() * 20L);
 		Bukkit.getServer().getLogger().info("DCCore has successfully been enabled!");
+		
+		//initialize Discord Webhook
+	        final DiscordWebhook web = new DiscordWebhook(this.getConfig().getString("Webhook"));
+       		final String message = "Listening";
+        	final DiscordWebhook.EmbedObject embed = new DiscordWebhook.EmbedObject();
+        	embed.addField(message, ":)", false);
+        	embed.setColor(Color.green);
+        	web.addEmbed(embed);
+        	try {
+        	    web.execute();
+        	}
+       		 catch (Exception exception) {
+         	   this.getLogger().warning("[Discord Vote Logger]Unable to send plugin load message to discord");
+       		}
```

Comment:
Why do we send an "initialized" message? Also, the whitespace is poorly formatted, it is hard to read